### PR TITLE
Release/version 3.2.0

### PR DESCRIPTION
**Pull request by M-Wong**

### CODE REVIEW

File: common/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -247,7 +247,7 @@ Dans tous les cas, vous devez aussi présenter une pièce d'identité sur demand
     <string name="wallet_faq_works_answer_2">"Oui. Vous pouvez par exemple stocker dans l'app les certificats COVID de tous les membres de votre famille. À noter également dans ce cas qu'un certificat COVID n'est valable que si son détenteur présente une pièce d'identité."</string>
 
     <!-- Antwort 8 im FAQ-Screen der Wallet App -->
-    <string name="wallet_faq_works_answer_3">"L'app contient la date d'expiration de votre certificat COVID en Suisse. En pressant le bouton prévu à cet effet, vous pouvez en outre vérifier dans la vue détaillée si votre certificat COVID est encore valable."</string>
+    <string name="wallet_faq_works_answer_3">"L’app verifica automaticamente se il certificato soddisfa i criteri di validità applicati in Svizzera. Se il certificato è scaduto o se non è valido per motivi tecnici, l’informazione compare direttamente nell’app."</string>
```
Comment (denysvitali) [inline common/src/main/res/values-fr/strings.xml:250]: This should be in french!

File: common/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -1445,31 +1445,31 @@ Code: {ERROR_CODE}"</string>
...
+    <string name="wallet_refresh_button_info_link_text">"Pour en savoir plus"</string>
 
     <!-- wallet refresh button removal info box link url -->
     <string name="wallet_refresh_button_info_link_url">"https://www.bag.admin.ch/bag/fr/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/covid-zertifikat/covid-zertifikat-pruefer-aussteller-technische-informationen.html#-1006172949"</string>
 
     <!-- text for info pop-up in wallet app describing 2G+ OK state -->
-    <string name="wallet_check_mode_info_2g_plus_ok_text">"Das Zertifikat ermöglicht den Zutritt zu Orten und Aktivitäten mit 2G+-Regelung. (Das Zertifikat für die Impfung oder Genesung ist weniger als 120 Tage alt.)"</string>
+    <string name="wallet_check_mode_info_2g_plus_ok_text">"L’accesso a strutture e manifestazioni è consentito alle persone per le quali sono trascorsi al massimo 120 giorni dal completamento del ciclo vaccinale, dalla somministrazione della dose di richiamo o dalla guarigione (sulla base di un test PCR). In questo caso non è necessario un ulteriore certificato di test."</string>
```
Comment (denysvitali) [inline common/src/main/res/values-fr/strings.xml:1466]: This should be in french!

File: common/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -1339,11 +1339,11 @@ Code: {ERROR_CODE}"</string>
 
     <!-- 👍 -->
     <!-- text for info pop-up in wallet app describing 2G OK state -->
-    <string name="wallet_check_mode_info_2g_ok_text">"Per l’accesso a strutture e a manifestazioni in cui si applica la regola 2G sono ammessi certificati COVID per persone vaccinate o guarite. I certificati COVID per persone testate negative e i certificati light non sono accettati."</string>
+    <string name="wallet_check_mode_info_2g_ok_text">"L’accesso  a strutture e manifestazioni è consentito alle persone vaccinate o guarite."</string>
```
Comment (denysvitali) [inline common/src/main/res/values-it/strings.xml:1342]: Is there a reason for the additional space here?

**Review by denysvitali — CHANGES_REQUESTED** [review]

**Review by benz-ubique — APPROVED** [review]

**Comment by denysvitali** [comment]
Please, check my review before merging: there are some italian texts in the french files.

File: common/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (benz-ubique) [reply common/src/main/res/values-fr/strings.xml:250]: This issue is fixed with the backend config request, this overrides this file content. We will ship this fallback file again for the release 3.3 with the correct language.

File: common/src/main/res/values-it/strings.xml (hunk 3 above)
Comment (benz-ubique) [reply common/src/main/res/values-it/strings.xml:1342]: No, will be fixed in the next release. Thanks for mentioning it.

File: common/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (benz-ubique) [reply common/src/main/res/values-fr/strings.xml:1466]: Thanks, we already noticed this in https://github.com/admin-ch/CovidCertificate-App-Android/issues/378. These strings are also part of the app config, so we already fixed it server-side https://github.com/admin-ch/CovidCertificate-App-Config-Service/pull/45 (the string in the apps are fallbacks if the app has no network connection).

This branch is what we release as v3.2.0 last week, so we will merge it as is. We will fix the strings in a later PR.

File: common/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (benz-ubique) [reply common/src/main/res/values-fr/strings.xml:250]: Thanks for noticing this. We will fix this in the next release on Monday.

**Pull request merged**